src/utilities/eicrecon/eicrecon_cli.cpp

```diff
@@ -320,8 +320,9 @@ void PrintPodioCollections(JApplication* app) {
       for (auto* event_source : event_sources) {
         //                    std::cout << event_source->GetPluginName() << std::endl;  // podio.so
         //                    std::cout << event_source->GetResourceName() << std::endl;
-        if (event_source->GetPluginName().find("podio") != std::string::npos)
-          event_source->DoInitialize();
+        if (event_source->GetPluginName().find("podio") != std::string::npos) {
+          event_source->DoOpen();
+        }
```

Comment:
Will this work in 2.3.1?
If we merge https://github.com/eic/EICrecon/pull/1735 that would be the earliest tested, according to https://github.com/eic/EICrecon/issues/1705#issuecomment-2655044372